routes/login/login.spec.js

```diff
@@ -372,12 +372,12 @@ describe('Test /login responses', () => {
       expect(response.statusCode).toBe(200)
     })
 
-    test('it returns a 422 response for no posted value', async () => {
+    test('it returns a 302 response for no posted value', async () => { // since 0 is what is displayed by default, a user could leave it blank and press enter. This might not be great in user testing scenario but should be a valid answer
       const response = await request(app).post('/login/auth')
-      expect(response.statusCode).toBe(422)
+      expect(response.statusCode).toBe(302)
```

Comment:
Interesting, this means that if the key is missing, it still works. 🤔 

Doesn't really matter for now, but something to keep in mind.

Reply:
Ya, it'll work because 0 is a valid response AND we don't have any validation on that value. When we add validation empty will mean 0 which will most likely not be the correct value.